13-location.Rmd

```diff
@@ -417,7 +417,7 @@ Finally, we `rbind` all shops into one large `sf` object.
 # select only specific columns
 shops = map(shops, dplyr::select, osm_id, shop)
 # putting all list elements into a single dataframe
-shops = do.call(shops, rbind)
+shops = do.call(rbind, shops)
```

Comment:
Heads-up @jannes-m just tried to reproduce the work and found the previous command to fail with:

```r
shops = do.call(shops, rbind)
# Error in do.call(shops, rbind) : second argument must be a list
```

Reply:
yep, this was a typo, thanks for fixing!